modules/getting-started-cli-load-data-output.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * getting-started/openshift-cli.adoc
+
+:_content-type: PROCEDURE
+[id="getting-started-cli-load-data-output_{context}"]
+
+= Loading data and displaying the national parks map
+
+You deployed the `parksmap` and `nationalparks` applications and then deployed the `mongodb-nationalparks` database. However, no data has been loaded _into_ the database.
+
+.Prerequisites
+
+* You must have access to an {product-title} cluster.
+* You must have installed the OpenShift CLI (`oc`).
+* You have a deployed image.
+
+.Procedure
+
+. To load national parks data, enter the following command:
++
+[source,terminal]
+----
+$ oc exec $(oc get pods -l component=nationalparks | tail -n 1 | awk '{print $1;}') -- curl -s http://localhost:8080/ws/data/load
+----
++
+.Example output
++
+[source,text]
+----
+"Items inserted in database: 2893"
+----
+
+. To verify that your data is loaded properly, enter the following command:
++
+[source,terminal]
+----
+oc exec $(oc get pods -l component=nationalparks | tail -n 1 | awk '{print $1;}') -- curl -s http://localhost:8080/ws/data/all
+----
++
+.Example output
+_Output trimmed for clarity:_
++
+[source,text]
```

Comment:
You use "source,text" here but "source,terminal" most other places. I think we've been wanting to move to source,text, but whichever you use for now should be consistent throughout the doc